vector/src/main/java/im/vector/app/features/spaces/SpaceListFragment.kt

```diff
@@ -71,6 +73,7 @@ class SpaceListFragment :
         homeActivitySharedActionViewModel = activityViewModelProvider[HomeSharedActionViewModel::class.java]
         roomListSharedActionViewModel = activityViewModelProvider[RoomListSharedActionViewModel::class.java]
         views.stateView.contentView = views.groupListView
+        views.spacesEmptyButton.onClick { onAddSpaceSelected() }
```

Comment:
This is maybe not new, but when the user clicks on this button, do we want to close the bottomsheet?

Reply:
@ericdecanini could you please verify?

Reply:
Ideally we don't want to close it immediately in case the user navigates back but we want to close it following a successful space creation. This may already work as it is but it would be good to verify this